console/src/main/java/org/web3j/console/project/Project.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import static org.web3j.codegen.Console.exitError;
+
+public class Project {
+
+    private Project(final Builder builder) {}
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+
+        private ProjectStructure projectStructure;
+        private TemplateProvider templateProvider;
+        private File solidityImportPath;
+
+        Builder() {}
```

Comment:
you don't need an empty contructor - should be using the `builder()` method above

Reply:
removed.